src/main/java/edu/wpi/grip/core/Operation.java

```diff
@@ -22,6 +22,10 @@
      */
     String getDescription();
 
+    default String getHyphenatedName(){
```

Comment:
If you want this somewhere else please offer a proposition as to where it should reside.